src/tools/recvfuzz/recvfuzz.cpp

```diff
@@ -148,13 +230,13 @@ struct TlsContext
             OnRecvQuicTP,
             NULL
         };
-        
+         
```

Comment:
nit: please make sure no lines have trailing spaces. You can update your editor to do this for you.